src/components/Editor/Editor.css

```diff
@@ -51,6 +51,22 @@ html[dir="rtl"] .editor-mount {
   direction: ltr;
 }
 
+:not(.empty-line):not(.new-breakpoint) > .CodeMirror-gutter-wrapper:hover {
+  width: 60px;
+  height: 13px;
+  left: -55px !important;
+  background-color: var(--theme-highlight-gray) !important;
+  mask: url(/images/breakpoint.svg) no-repeat;
+  mask-size: 100%;
+  mask-position: 0 1px;
+}
+
+:not(.empty-line):not(.new-breakpoint) > .CodeMirror-gutter-wrapper:hover > .CodeMirror-linenumber {
+  left: auto !important;
+  right: 6px;
+  color: var(--theme-body-color);
```

Comment:
This looks good on light theme, but on dark theme it's hard to read. Is there a color that might work well for this?

Reply:
I love the detail in testing the dark theme too! Great work!  👍 

How about `--theme-body-color-alt` for the dark theme?  Seem reasonable to me :)